<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>CMM 2017 | Bader Lab</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="description" content="CMM-2017 Syllabus Applied Math and Statistics 550.450 Computational Molecular Medicine Spring 2017 (4 credits, EN)  Description Computational systems biology has emerged as the dominant framework for analyzing high-dimensional “omics” data in order to uncover the relationships among molecules, networks and disease. In particular, many of the core methodologies are based on statistical modeling, including machine learning, stochastic processes and statistical inference. We will cover the key aspects of this methodology, including measuring associations, testing multiple hypotheses, and learning predictors, Markov chains and graphical models.">
    <meta name="generator" content="Hugo 0.91.2" />
    
    
    
    
      <meta name="robots" content="noindex, nofollow">
    

    
<link rel="stylesheet" href="/ananke/css/main.min.css" >



    
    
    
      

    

    
    
    <meta property="og:title" content="CMM 2017" />
<meta property="og:description" content="CMM-2017 Syllabus Applied Math and Statistics 550.450 Computational Molecular Medicine Spring 2017 (4 credits, EN)  Description Computational systems biology has emerged as the dominant framework for analyzing high-dimensional “omics” data in order to uncover the relationships among molecules, networks and disease. In particular, many of the core methodologies are based on statistical modeling, including machine learning, stochastic processes and statistical inference. We will cover the key aspects of this methodology, including measuring associations, testing multiple hypotheses, and learning predictors, Markov chains and graphical models." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://baderzone.github.io/teaching/cmm-2017/" /><meta property="article:section" content="teaching" />
<meta property="article:published_time" content="2022-03-25T12:59:06-04:00" />
<meta property="article:modified_time" content="2022-03-25T12:59:06-04:00" />

<meta itemprop="name" content="CMM 2017">
<meta itemprop="description" content="CMM-2017 Syllabus Applied Math and Statistics 550.450 Computational Molecular Medicine Spring 2017 (4 credits, EN)  Description Computational systems biology has emerged as the dominant framework for analyzing high-dimensional “omics” data in order to uncover the relationships among molecules, networks and disease. In particular, many of the core methodologies are based on statistical modeling, including machine learning, stochastic processes and statistical inference. We will cover the key aspects of this methodology, including measuring associations, testing multiple hypotheses, and learning predictors, Markov chains and graphical models."><meta itemprop="datePublished" content="2022-03-25T12:59:06-04:00" />
<meta itemprop="dateModified" content="2022-03-25T12:59:06-04:00" />
<meta itemprop="wordCount" content="828">
<meta itemprop="keywords" content="" /><meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="CMM 2017"/>
<meta name="twitter:description" content="CMM-2017 Syllabus Applied Math and Statistics 550.450 Computational Molecular Medicine Spring 2017 (4 credits, EN)  Description Computational systems biology has emerged as the dominant framework for analyzing high-dimensional “omics” data in order to uncover the relationships among molecules, networks and disease. In particular, many of the core methodologies are based on statistical modeling, including machine learning, stochastic processes and statistical inference. We will cover the key aspects of this methodology, including measuring associations, testing multiple hypotheses, and learning predictors, Markov chains and graphical models."/>

	
  </head>

  <body class="ma0 georgia bg-near-white">

    
   
  

  
  
  <header class="cover bg-top" style="background-image: url('https://baderzone.github.io/images/research_cancer_thumbnails.jpg');">
    <div class="bg-black-60">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l justify-between items-center center">
    <a href="/" class="f3 fw2 hover-white no-underline white-90 dib">
      
        Bader Lab
      
    </a>
    <div class="flex-l items-center">
      

      
        <ul class="pl0 mr3">
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/members/" title="Members page">
              Members
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/publications/" title="Publications page">
              Publications
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/research/" title="Research page">
              Research
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/teaching/" title="Teaching page">
              Teaching
            </a>
          </li>
          
        </ul>
      
      
<div class="ananke-socials">
  
</div>
    </div>
  </div>
</nav>

      <div class="tc-l pv6 ph3 ph4-ns">
        
          <h1 class="f2 f1-l fw2 white-90 mb0 lh-title">CMM 2017</h1>
          
        
      </div>
    </div>
  </header>



    <main class="pb7" role="main">
      
  
  <article class="flex-l flex-wrap justify-between mw8 center ph3">
    <header class="mt4 w-100">
      <aside class="instapaper_ignoref b helvetica tracked">
          
        TEACHING
      </aside>
      










  <div id="sharing" class="mt3 ananke-socials">
    
  </div>


      <h1 class="f1 athelas mt3 mb1">CMM 2017</h1>
      
      
      
      <time class="f6 mv4 dib tracked" datetime="2022-03-25T12:59:06-04:00">March 25, 2022</time>
      

      
      
    </header>
    <div class="nested-copy-line-height lh-copy serif f4 nested-links mid-gray pr4-l w-two-thirds-l"><h2 id="cmm-2017">CMM-2017</h2>
<h4 id="syllabus">Syllabus</h4>
<h4 id="applied-math-and-statistics-550450">Applied Math and Statistics 550.450</h4>
<h4 id="computational-molecular-medicine">Computational Molecular Medicine</h4>
<h4 id="spring-2017">Spring 2017</h4>
<h4 id="4-credits-en">(4 credits, EN)</h4>
<hr>
<h4 id="description">Description</h4>
<p>Computational systems biology has emerged as the dominant framework for analyzing high-dimensional “omics” data in order to uncover the relationships among molecules, networks and disease. In particular, many of the core methodologies are based on statistical modeling, including machine learning, stochastic processes and statistical inference. We will cover the key aspects of this methodology, including measuring associations, testing multiple hypotheses, and learning predictors, Markov chains and graphical models. In addition, by studying recent important articles in cancer systems biology, we will illustrate how this approach enhances our ability to annotate genomes, discover molecular disease networks, detect disease, predict clinical outcomes, and characterize disease progression. Whereas a good foundation in probability and statistics is necessary, no prior exposure to molecular biology is required (although helpful).</p>
<h4 id="instructor">Instructor</h4>
<p>Prof. Joel S. Bader, joel.bader at jhu.edu, <a href="www.baderzone.org">www.baderzone.org</a></p>
<p>Course developed by Prof. Donald Geman</p>
<p>Office: Clark 201C; Miller Research Building 319</p>
<p>Office hours: 5:45 PM Shaffer 303, with discussions continued at One World Cafe</p>
<h4 id="teaching-assistants">Teaching Assistants</h4>
<ul>
<li>Tobi Bosede, abosede2 at jhu.edu</li>
<li>Kamel Lahouel, klahoue1 at jhu.edu</li>
</ul>
<h4 id="meetings">Meetings</h4>
<p>Lectures: Monday and Wednesday, 4:30 – 5:45 PM, Shaffer 303</p>
<p>Sections</p>
<ol>
<li>Section 2, Friday, 9:00 – 9:50 AM, Hodson 216, Kamel</li>
<li>Section 1, Friday, 1:30 PM – 2:20 PM, Gilman 377, Tobi</li>
</ol>
<p>TA Office Hours</p>
<ul>
<li>Mondays, 1-2 PM, Whitehead 212</li>
<li>Fridays, 10-11 AM, Clark 307B</li>
</ul>
<h4 id="key-dates">Key Dates</h4>
<p>Lecture, HW, Quiz, and Exam Dates</p>
<p>Midterm, Monday, March 27
Final exam, according to registrar schedule</p>
<p>Syllabus with dates:</p>
<ul>
<li><a href="/files/cmm_syllabus_2017_02_06.pdf">cmm_syllabus_2017_02_06</a></li>
<li><a href="/files/cmm_syllabus_2017_02_19.pdf">cmm_syllabus_2017_02_19</a></li>
</ul>
<h4 id="textbook">Textbook</h4>
<p>Lecture notes posted on BlackBoard.</p>
<h4 id="homeworks">Homeworks</h4>
<p>Posted on BlackBoard.</p>
<h4 id="on-line-material">On-line material</h4>
<p>Posted on BlackBoard.</p>
<h4 id="prerequisites">Prerequisites</h4>
<p>EN.550.420 and EN.550.430 (Introduction to Probability and Introduction to Statistics)</p>
<p>All prerequisites may be satisfied by an equivalent course or by the instructor’s permission.</p>
<h4 id="online-resources">Online Resources</h4>
<p>BlackBoard? Yes, BlackBoard.</p>
<h4 id="course-objectives">Course Objectives</h4>
<ol>
<li>Have a working knowledge of the modern understanding of genetics and genomics.</li>
<li>Understand probability and statistics in the small n, large p limit (many more variables than observations).</li>
<li>Understand how to perform hypothesis testing with applications to genomic data.</li>
<li>Understand how to learn and apply classifiers using biological data as input.</li>
<li>Develop and apply stochastic models for biological processes.</li>
<li>Develop and apply graphical models to describe latent variables in biological systems.</li>
</ol>
<h4 id="course-topics">Course Topics</h4>
<ol>
<li>Basic genetics and genomics, Central Dogma, biological information and high-throughput data</li>
<li>Genotype and phenotype, statistical genetics</li>
<li>Hypothesis testing for DNA (GWAS) and RNA (differential expression)</li>
<li>Cancer biology, cancer classification, driver mutations</li>
<li>Tumor evolution, Galton-Watson models, epigenetics</li>
<li>Graphical models, hidden Markov models for biological sequences, signaling networks</li>
</ol>
<h4 id="course-expectations--grading">Course Expectations &amp; Grading</h4>
<p>6 homework assignments, each worth 1 grading unit</p>
<p>Midterm exam, worth 2 grading units</p>
<p>Final exam, worth 3 grading units</p>
<p>All work (including HW and exams) should be independent.</p>
<p>If you must be absent for an academic or professional reason, including conferences, medical school or graduate school interviews, or other on-site interviews, please speak to me ahead of time. You must speak to me ahead of time if an absence conflicts with an exam date.</p>
<p>If you are absent for a non-academic reason, primarily health-related, please have the Dean of Students email me on your behalf.</p>
<h4 id="ethics">Ethics</h4>
<p>The strength of the university depends on academic and personal integrity. In this course, you must be honest and truthful. Ethical violations include cheating on exams, plagiarism, reuse of assignments, improper use of the Internet and electronic devices, unauthorized collaboration, alteration of graded assignments, forgery and falsification, lying, facilitating academic dishonesty, and unfair competition.</p>
<p>In addition, the specific ethics guidelines for this course are:</p>
<p>Homework: students should work independently and eschew use of previous years’ assignments in preparing homework assignments for this course.
Late homework will receive a 50% deduction.
Study materials: all study materials are generally permitted, including course materials from previous years. Advance copies of current-year questions are not permitted, however. Students taking exams at different times, particularly for make-up exams, may not share information.
Exams: only blank paper, pens, and pencils are permitted. Electronic devices, including calculators, phones, and computers, are strictly forbidden. Equation sheets, if necessary, will be provided in the exam itself.
Report any violations you witness to the instructor.</p>
<p>You can find more information about university misconduct policies on the web at these sites:</p>
<p><a href="e-catalog.jhu.edu/undergrad-students/student-life-policies">Undergraduates</a><br>
<a href="e-catalog.jhu.edu/undergrad-students/student-life-policies">Graduate students</a></p>
<h4 id="students-with-disabilities">Students with Disabilities</h4>
<p>Any student with a disability who may need accommodations in this class must obtain an accommodation letter from Student Disability Services, 385 Garland, (410) 516-4720,</p>
<p><a href="studentdisabilityservices@jhu.edu">studentdisabilityservices@jhu.edu</a></p>
<h4 id="abet-outcomes">ABET Outcomes</h4>
<ul>
<li>Ability to apply mathematics, science and engineering principles (a).</li>
<li>Ability to design and conduct experiments, analyze and interpret data (b).</li>
<li>Ability to identify, formulate and solve engineering problems (e).</li>
<li>Understanding of professional and ethical responsibility (f).</li>
<li>Ability to communicate effectively (g).</li>
<li>The broad education necessary to understand the impact ofengineering solutions in a global and societal context (h).</li>
<li>Recognition of the need for and an ability to engage in life-long learning (i).</li>
<li>Knowledge of contemporary issues (j).</li>
<li>Ability to use the techniques, skills and modern engineeringtools necessary for engineering practice (k).</li>
</ul>
<ul class="pa0">
  
</ul>
<div class="mt6 instapaper_ignoref">
      
      
      </div>
    </div>

    <aside class="w-30-l mt6-l">




</aside>

  </article>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href="https://baderzone.github.io/" >
    &copy;  Bader Lab 2022 
  </a>
    <div>
<div class="ananke-socials">
  
</div></div>
  </div>
</footer>

  </body>
</html>
