<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Teaching | Bader Lab</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="description" content="">
    <meta name="generator" content="Hugo 0.91.2" />
    
    
    
    
      <meta name="robots" content="noindex, nofollow">
    

    
<link rel="stylesheet" href="/ananke/css/main.min.css" >



    
    
    
      

    

    
    
      <link href="/teaching/index.xml" rel="alternate" type="application/rss+xml" title="Bader Lab" />
      <link href="/teaching/index.xml" rel="feed" type="application/rss+xml" title="Bader Lab" />
      
    
    
    <meta property="og:title" content="Teaching" />
<meta property="og:description" content="" />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://baderzone.github.io/teaching/" />

<meta itemprop="name" content="Teaching">
<meta itemprop="description" content=""><meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Teaching"/>
<meta name="twitter:description" content=""/>

	
  </head>

  <body class="ma0 georgia bg-near-white">

    

  
  
  <header class="cover bg-top" style="background-image: url('https://baderzone.github.io/images/research_cancer_thumbnails.jpg');">
    <div class="bg-black-60">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l justify-between items-center center">
    <a href="/" class="f3 fw2 hover-white no-underline white-90 dib">
      
        Bader Lab
      
    </a>
    <div class="flex-l items-center">
      

      
        <ul class="pl0 mr3">
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/members/" title="Members page">
              Members
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/publications/" title="Publications page">
              Publications
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/research/" title="Research page">
              Research
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/teaching/" title="Teaching page">
              Teaching
            </a>
          </li>
          
        </ul>
      
      
<div class="ananke-socials">
  
</div>
    </div>
  </div>
</nav>

      <div class="tc-l pv4 pv6-l ph3 ph4-ns">
        <h1 class="f2 f-subheadline-l fw2 white-90 mb0 lh-title">
          Teaching
        </h1>
        
      </div>
    </div>
  </header>


    <main class="pb7" role="main">
      
  <article class="pa3 pa4-ns nested-copy-line-height">
    <section class="cf ph3 ph5-l pv3 pv4-l f4 tc-l center measure-wide lh-copy mid-gray"><h2 id="teaching">Teaching</h2>
<h3 id="currently-taught">Currently Taught</h3>
<h4 id="580248-systems-biology-of-the-cell">580.248 Systems Biology of the Cell</h4>
<p>Living systems respond to their environments. Understanding cellular responses is important for diseases caused by aberrant responses and for engineering of new life with desired properties. This course develops quantitative models for cellular response functions. It begins with linear response models for protein signal transduction, including an introduction to complex analysis and contour integrals and examination of the MAP kinase signaling cascade. Laplace transforms are related to characteristic functions and moment-generating functions from probability theory. Next, non-linear effects from saturation and cooperative are introduced in the context of transcriptional regulation. Continuous models are related to stochastic dynamics for small numbers of regulatory proteins, with applications to Delta-Notch signaling and lateral inhibition. Finally, diffusion-based models are introduced for long-range patterning and morphogenesis.</p>
<h3 id="previously-taught">Previously Taught</h3>
<h4 id="550450-computational-molecular-medicine-department-of-applied-math-and-statistics">550.450 Computational Molecular Medicine (Department of Applied Math and Statistics)</h4>
<p>This course was developed by Prof. Don Geman and Prof. Laurent Younes in the Department of Applied Math and Statistics. It introduces machine learning methods used in biomedicine, begins with a foundation in information theory, uses the Kullback-Liebler divergence as motivation for likelihood ratio tests, considers aspects of statistical testing relevant for multiple testing in high-dimensional spaces, and then provides examples of classification and regression problems encountered in genomics.</p>
<h4 id="580420-build-a-genome-also-020420-in-department-of-biology">580.420 Build-a-Genome (also 020.420 in Department of Biology)</h4>
<p>This unique course was a cross between a synthetic biology research lab and life in a biotech company. Students were trained in the basics of synthetic biology and then given DNA oligomers and reagents to synthesize building blocks for a synthetic yeast chromosome as part of the Sc2.0 project. Students had keys to the lab for 24/7 access and had weekly lab meetings to present and discuss results and troubleshoot.</p>
<p>This course is no longer offered because it was too successful: all the synthesis for the synthetic yeast genome has been completed, and synthetic chromosomes are in various stages of debugging. Versions of this course have been taught around the world, and similar courses continue to be taught by former teaching assistants.</p>
<h4 id="580429-systems-bioengineering-3">580.429 Systems Bioengineering 3</h4>
<p>This full-semester course was the origin of Systems Biology of the Cell. It included additional material on simulating stochastic systems with the Gillespie algorithm, stability analysis and its relationship with transition state theory in chemical kinetics, and biological networks including flux-balance analysis, graph diffusion kernels, network motifs, and stochastic block models.</p>
</section>
    <section class="flex-ns flex-wrap justify-around mt5">
      
        <div class="relative w-100 w-30-l mb4 bg-white">
          <div class="relative w-100 mb4 bg-white nested-copy-line-height">
  <div class="bg-white mb3 pa4 gray overflow-hidden">
    <span class="f6 db">Teaching</span>
    <h1 class="f3 near-black">
      <a href="/teaching/sboc-2020/" class="link black dim">
        SBOC 2020
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      SBOC-2020 Syllabus Biomedical Engineering 580.248 Systems Biology of the Cell Spring 2020 (2 credits, EN) URL  Main Lecture: See Blackboard Discussion Sections: See Blackboard  Perusall Code  Make yourself an account at https://www.perusall.com/ Enroll with this course code: BADER-RYZDC  Syllabus with dates  sboc-syllabus-2020-03-24  Description Systems Biology of the Cell (580.429 SB3) is a half-semester course required of all BME majors. Progress in biomedicine requires understanding how individual cellular components function together.
    </div>
  </div>
</div>

        </div>
      
        <div class="relative w-100 w-30-l mb4 bg-white">
          <div class="relative w-100 mb4 bg-white nested-copy-line-height">
  <div class="bg-white mb3 pa4 gray overflow-hidden">
    <span class="f6 db">Teaching</span>
    <h1 class="f3 near-black">
      <a href="/teaching/cmm-2017/" class="link black dim">
        CMM 2017
      </a>
    </h1>
    <div class="nested-links f5 lh-copy nested-copy-line-height">
      CMM-2017 Syllabus Applied Math and Statistics 550.450 Computational Molecular Medicine Spring 2017 (4 credits, EN)  Description Computational systems biology has emerged as the dominant framework for analyzing high-dimensional “omics” data in order to uncover the relationships among molecules, networks and disease. In particular, many of the core methodologies are based on statistical modeling, including machine learning, stochastic processes and statistical inference. We will cover the key aspects of this methodology, including measuring associations, testing multiple hypotheses, and learning predictors, Markov chains and graphical models.
    </div>
  </div>
</div>

        </div>
      
    </section></article>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href="https://baderzone.github.io/" >
    &copy;  Bader Lab 2022 
  </a>
    <div>
<div class="ananke-socials">
  
</div></div>
  </div>
</footer>

  </body>
</html>
