<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
    
    <title>Members | Bader Lab</title>
    <meta name="viewport" content="width=device-width,minimum-scale=1">
    <meta name="description" content="">
    <meta name="generator" content="Hugo 0.91.2" />
    
    
    
    
      <meta name="robots" content="noindex, nofollow">
    

    
<link rel="stylesheet" href="/ananke/css/main.min.css" >



    
    
    
      

    

    
    
      <link href="/members/index.xml" rel="alternate" type="application/rss+xml" title="Bader Lab" />
      <link href="/members/index.xml" rel="feed" type="application/rss+xml" title="Bader Lab" />
      
    
    
    <meta property="og:title" content="Members" />
<meta property="og:description" content="" />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://baderzone.github.io/members/" />

<meta itemprop="name" content="Members">
<meta itemprop="description" content=""><meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Members"/>
<meta name="twitter:description" content=""/>

	
  </head>

  <body class="ma0 georgia bg-near-white">

    

  
  
  <header class="cover bg-top" style="background-image: url('https://baderzone.github.io/images/research_cancer_thumbnails.jpg');">
    <div class="bg-black-60">
      <nav class="pv3 ph3 ph4-ns" role="navigation">
  <div class="flex-l justify-between items-center center">
    <a href="/" class="f3 fw2 hover-white no-underline white-90 dib">
      
        Bader Lab
      
    </a>
    <div class="flex-l items-center">
      

      
        <ul class="pl0 mr3">
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/members/" title="Members page">
              Members
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/publications/" title="Publications page">
              Publications
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/research/" title="Research page">
              Research
            </a>
          </li>
          
          <li class="list f5 f4-ns fw4 dib pr3">
            <a class="hover-white no-underline white-90" href="/teaching/" title="Teaching page">
              Teaching
            </a>
          </li>
          
        </ul>
      
      
<div class="ananke-socials">
  
</div>
    </div>
  </div>
</nav>

      <div class="tc-l pv4 pv6-l ph3 ph4-ns">
        <h1 class="f2 f-subheadline-l fw2 white-90 mb0 lh-title">
          Members
        </h1>
        
      </div>
    </div>
  </header>


    <main class="pb7" role="main">
      
  <article class="pa3 pa4-ns nested-copy-line-height">
    <section class="cf ph3 ph5-l pv3 pv4-l f4 tc-l center measure-wide lh-copy mid-gray"><h2 id="members">Members</h2>
<h3 id="joel-s-bader">Joel S. Bader</h3>
<p><img src="/images/members/members_jsb.jpg" alt="Joel S. Bader"></p>
<p>Joel S. Bader
Homewood:
Wyman Park Building 407W,
Johns Hopkins University,
3400 North Charles Street,
Baltimore, MD 21218</p>
<p>School of Medicine:
Miller Research Building 319,
Johns Hopkins School of Medicine,
733 North Broadway,
Baltimore, MD 21205
Tel. 410-516-7417</p>
<p><a href="mailto:joel.bader@jhu.edu">joel.bader@jhu.edu</a></p>
<p>Joel S. Bader, Ph.D., professor in the Department of Biomedical Engineering, uses engineering-based mathematical, computational, and high-throughput methods to study biomedical systems. Bader is Interim Director of the High-Throughput Biology Center at the School of Medicine, has a secondary appointment in the Department of Computer Science, and is a member of the Institute of Computational Medicine (School of Engineering) and the Institute of Genetic Medicine (School of Medicine).</p>
<p>Bader’s research interest is the relationship between genotype, the DNA sequence that encodes life’s information, and phenotype, a living system’s observable properties. Applications are to human disease, primarily complex genetic disorders and cancer, and to designed synthetic systems. The Human Genome Project, which provided a reference sequence of the 3 billion DNA letters that are the instructions for human life, also identified heritable genetic variants that influence disease risk. Mutations that arise in individual cells similarly can lead to cancer and other diseases. Bader develops new computational methods and joint wet-lab approaches to define how inborn genetic variants and acquired mutations lead to disease, with the goal of developing new therapies.</p>
<p>Bader leads the Johns Hopkins Cancer Target Discovery and Development (CTD2) Center, which was funded by the National Cancer Institute to identify new targets for metastatic breast cancer. His study is recruiting women receiving breast cancer treatment at Johns Hopkins Hospital to donate biological specimens that will help reveal the molecular drivers of tumor invasion and metastasis. These phenotypes, rather than cell proliferation in the primary tumor, are the main cause of breast cancer mortality.</p>
<p>In synthetic biology, Bader’s lab develops methods for designing, building, testing, and analyzing DNA sequences that encode genes, pathways, chromosomes, and entire genomes. He is the computational leader of the international Saccharomyces cerevisiae 2.0 (Sc2.0) project, whose aim is to create a yeast cell with a synthetic genome. The milestone of synthesizing 6 of the 16 yeast chromosomes was highlighted in a series of 7 papers in the March 10, 2017, issue of the journal Science.Bader’s lab also develops technologies for biosafety and biosecurity, preventing escape of engineered life and identifying evidence of genetic engineering in genetic sequence data.</p>
<p>Bader developed and teaches a core course in systems biology in the BME undergraduate program and has also been faculty for courses in the Department of Applied Math and Statistics and the Department of Biology. He is a faculty advisor to the Tau Beta Pi Maryland Alpha Chapter at Johns Hopkins. Bader is a member of the Climate, Culture, and Campus Experience Subcommittee of the Homewood Council on Inclusive Excellence and has served on the University-wide Library Advisory Council. He is a Fellow of the American Institute for Medical and Biological Engineering (AIMBE); a member of the NIH Genomics, Computational Biology and Technology (GCAT) study section; and Deputy Editor of PLOS Computational Biology.</p>
<p>Prior to joining Johns Hopkins, Bader was Director of Bioinformatics at CuraGen Corp., where he worked from 1995-2003 and co-invented the 454 Genome Sequencer, the first next-generation DNA sequencer to reach the market.</p>
<p>Bader received a B.S. in Biochemistry from Lehigh University, where he was elected to Phi Beta Kappa and Tau Beta Pi, and a Ph.D. in Chemistry from the University of California, Berkeley, where he was an NSF Pre-Doctoral Fellow. He did postdoctoral research at Columbia University. He joined the faculty of the Whiting School of Engineering as an assistant professor in 2003, which has been his academic home ever after.</p>
<p>Fun facts: Joel has 6 kids, 2 dogs, and has run marathons in San Francisco, New York City, Washington’s Crossing, and Jerusalem.</p>
<h3 id="current-members-alphabetical-by-last-name">Current Members (alphabetical by last name)</h3>
<h4 id="yuchen-peter-ge">Yuchen (Peter) Ge</h4>
<p><a href="mailto:yge15@jhu.edu">yge15@jhu.edu</a></p>
<p>Yuchen Ge is a research assistant in the Bader lab while he pursues his M.S. in the Department of Biomedical Engineering at Johns Hopkins. Yuchen is relating gene expression patterns we observe in experimental data generated at Johns Hopkins with larger public databases of cancer gene expression to identify possible markers for more aggressive vs. less aggressive forms of breast cancer.</p>
<h4 id="aditya-kaushik">Aditya Kaushik</h4>
<p><a href="mailto:akaushi8@jhu.edu">akaushi8@jhu.edu</a></p>
<p>Aditya Kaushik is an undergraduate researcher in the Bader lab who is pursuing his B.S. in Chemical and Biomolecular Engineering at Johns Hopkins. He was a high school research intern at the Califano lab at Columbia University, and he is applying methods he learned there to similar problems in cancer gene regulation in the Bader lab.</p>
<p>Fun facts: Aditya and Joel have similar commuting patterns.</p>
<h4 id="hildur-knutsdottir">Hildur Knutsdottir</h4>
<p><img src="/images/members/members_hildur_knutsdottir.jpg" alt="Hildur Knutsdottir"></p>
<p>Hildur Knutsdottir<br>
<a href="mailto:hknutsd1@jhu.edu">hknutsd1@jhu.edu</a></p>
<p>Hildur Knustdottir, Ph.D., is a postdoctoral researcher in the Bader lab. She trained previously with Prof. Edelstein-Keshet at the University of British Columbia. Hildur is leading the computational analysis of human tumor specimens to identify drivers of phenotypes related to metastatic breast cancer.</p>
<p>Fun facts: Hildur has been first author and corresponding author on several of the papers to come out of her Ph.D. thesis.</p>
<h4 id="will-matern">Will Matern</h4>
<p><img src="/images/members/will_matern.jpg" alt="Will Matern"></p>
<p>Will Matern<br>
<a href="mailto:wmatern1@jhu.edu">wmatern1@jhu.edu</a></p>
<p>Will Matern is a Ph.D. candidate in the Department of Biomedical Engineering. He wants to understand how to shorten the duration and improve the efficacy of therapies for infectious disease and has been working on Mycobacterium tuberculosis, the causative pathogen of the disease tuberculosis, and other Mycobacteria including M. avium, which is an emerging pathogen in the U.S. Will has been co-mentored by Petros Karakousis, M.D., a long-time collaborator of the Bader lab, to learn wet-lab techniques that complement is computational and mathematical training.</p>
<p>Fun facts: Will has successfully written a postdoctoral proposal that will permit him to perform computational analysis of transposon library mutagenesis screens that he has been performing as a Ph.D. student. He is about to graduate with his Ph.D. and commutes on a bicycle.</p>
<h4 id="ashton-omdahl">Ashton Omdahl</h4>
<p><img src="/images/members/ashton-omdahl-150x150.jpg" alt="Ashton Omdahl"></p>
<p><a href="mailto:aomdahl1@jhmi.edu">aomdahl1@jhmi.edu</a></p>
<p>Ashton Omdahl is a rotation student in the Bader lab. He is a Ph.D. student in the Department of Biomedical Engineering and also in the Pre-Doctoral Training Program of the Institute of Computational Medicine. Ashton is using computational methods from gene and protein evolution and from natural language processing to identify patterns of research in synthetic biology. This project was inspired by work the group is doing to curate published examples of synthetic biology design to create classifiers for natural-vs-engineered sequences.</p>
<p>Fun facts: Ashton competes in triathlons, is a member of the JHU running club, and is webmaster for BME Edge.</p>
<h4 id="xi-peng">Xi Peng</h4>
<p><a href="mailto:menghanpx@gmail.com">menghanpx@gmail.com</a></p>
<p>Xi Peng, M.D., is a postdoctoral researcher in the Bader lab. Her medical speciality is ophthalmology. Her research focus in the Bader lab is the genetic basis for neuropsychiatric disorders, a project being done jointly with Dimitri Avramopoulos, M.D., Ph.D., in the Institute of Genetic Medicine at Johns Hopkins School of Medicine. Xi is developing and applying methods to predict the mechanism by which risk-enhancing variants identified by genome-wide association studies have their effects.</p>
<p>Fun facts: Xi trained in China and learned about U.S. culture with a book that is surprisingly accurate.</p>
<h4 id="matthew-perrone">Matthew Perrone</h4>
<p><img src="/images/members/members_matt_perrone.jpg" alt="Matthew Perrone"></p>
<p>Matthew Perrone<br>
<a href="mailto:mperron2@jhu.edu">mperron2@jhu.edu</a></p>
<p>Matthew Perrone is a Ph.D. candidate in the Department of Biomedical Engineering. He is developing network-based algorithms to prioritize druggable target proteins to interfere with disease-related processes caused by non-druggable driver genes and proteins.</p>
<p>Fun facts: Yes, Matt is really that happy. He’s Canadian.</p>
<h4 id="jonathan-prokos">Jonathan Prokos</h4>
<p><img src="/images/members/jonathan-prokos-7317.jpg" alt="Jonathan Prokos"></p>
<p>Jonathan Prokos<br>
<a href="mailto:jprokos1@jhu.edu">jprokos1@jhu.edu</a></p>
<p>Jonathan Prokos is an undergraduate researcher in the Bader lab who is pursuing a B.S. in the Department of Computer Science. He is working on methods to detect evidence of genetic engineering in DNA sequence data. Part of his project involves introducing watermarks into DNA sequence by using synonymous recoding to create unique sequence tags in designed sequences. These sequences are used by our collaborators as training examples for their natural-vs-engineered sequence classifiers.</p>
<p>Fun facts: Jonathan is a baseball catcher.</p>
<h4 id="yohannes-tsehay">Yohannes Tsehay</h4>
<p><img src="/images/members/members_yohannes_tsehay.jpg" alt="Yohannes Tsehay"></p>
<p>Yohannes Tsehay<br>
<a href="mailto:ytsehay1@jhmi.edu">ytsehay1@jhmi.edu</a></p>
<p>Yohannes Tsehay is a M.D. student at the Johns Hopkins School of Medicine who is performing a scholarly concentration in the Bader lab. He has developed methods for quantitative imaging that we are applying to identify molecular biomarkers for invasion, in particular abundance and distribution of proteins within organoids.</p>
<p>Fun facts: Yohannes was a physics major as an undergraduate, which is good because his research involves lots of Fourier transforms and convolution kernels to identify boundary pixels.</p>
<h4 id="manisha-warrier">Manisha Warrier</h4>
<p><a href="mailto:mwarrie1@jhu.edu">mwarrie1@jhu.edu</a></p>
<p>Manisha Warrier is a research assistant in the Bader lab while she completes her M.S. in Bioinformatics at Johns Hopkins. Manisha is building RNA-Seq analysis pipeline to study gene expression in organoids generated from genetically engineered mouse models of cancer and from human tissues.</p>
<h4 id="lucy-zhang">Lucy Zhang</h4>
<p><img src="/images/members/members_lucy_zhang.jpg.webp" alt="Lucy Zhang"></p>
<p>Lucy Zhang<br>
<a href="mailto:lzhan115@jhu.edu">lzhan115@jhu.edu</a></p>
<p>Lucy Zhang is an undergraduate researcher in the Bader lab while she pursues her B.S. in Computer Science at Johns Hopkins University. Lucy works on detecting evidence of genetic engineering in DNA sequences.</p>
<h3 id="alumni-alphabetical-by-last-name">Alumni (alphabetical by last name)</h3>
<h4 id="jitong-sophie-cai">Jitong (Sophie) Cai</h4>
<p>Jitong Cai is a Ph.D. candidate in the Department of Biomedical Engineering. She was a rotation student in the Bader lab 2018-2019 and worked on methods to analyze the genetics of known drug targets and to prioritize candidate genes from network data.</p>
<p>Fun facts: Jitong is being sought for rotations by many groups. We hope that after her rotations she returns to our lab!</p>
<h4 id="hailiang-huang">Hailiang Huang</h4>
<p><img src="/images/members/hailiang.jpg" alt="Hailiang Huang"></p>
<p>Hailiang Huang<br>
<a href="https://scholar.harvard.edu/huang">https://scholar.harvard.edu/huang</a></p>
<p>Hailiang Huang, Ph.D., is an Instructor in the Analytic and Translational Genetics Unit at Massachusetts General Hospital and an Associated Scientist of the Broad Institute of MIT and Harvard. He is also an instructor in the Department of Medicine at Harvard Medical School. Hailiang’s research focuses on the genetics of complex disorders, especially autoimmune and psychiatric disorders. He is interested in developing new statistical and analytical methods, and use them to pinpoint and understand the genetic factors driving human complex disorders. His studies usually use large-scale omics data from various consortia and public available sources such as UK Biobank, NIH Roadmap, and GTEx. Hailiang’s work in the Bader lab involved substantial collaborations with the Arking lab at the Institute of Genetic Medicine to develop new Bayesian frameworks for genome-wide association studies, including gene-based tests and other advances that we are still converting from figures to papers.</p>
<p>Fun facts: Hailiang heard about Johns Hopkins from his mother, Dr. Huang, who served as President of the Johns Hopkins campus in Nanjing, China.</p>
<h4 id="elisa-pappalardo">Elisa Pappalardo</h4>
<p><img src="/images/members/elisa.jpg" alt="Elisa Pappalardo"></p>
<p>Elisa Pappalardo</p>
<p>Elisa Pappalardo, Ph.D., works in scientific publishing and was a postdoctoral researcher in the Bader lab. She was the first Bader lab trainee to work on joint projects with the Ewald lab, and her initial efforts have become a major focus of the lab’s research.</p>
<p>Fun facts: Elisa and her husband were students together in Sicily.</p>
<h4 id="yongjin-park">Yongjin Park</h4>
<p><img src="/images/members/yongjin.jpg" alt="Yongjin Park"></p>
<p>Yongjin Park</p>
<p>Yongjin Park, Ph.D., is a postdoctoral researcher at the Broad Institute working on regulatory genomics and causal inference. His work in the Bader lab focused on generative models for biological networks, primarily stochastic block models for joint analysis of heterogeneous data. He developed elegant extensions of stochastic block models to dynamics networks, with gene and protein dynamics inferred from RNA-Seq time-course data.</p>
<h4 id="scott-patterson">Scott Patterson</h4>
<p><img src="/images/members/scott_patterson.jpg" alt="Scott Patterson"></p>
<p>Scott Patterson<br>
<a href="https://notablelabs.com">https://notablelabs.com</a></p>
<p>Scott Patterson is V.P. of Engineering at Notable Labs, a company founded to improve cancer therapy selection. He was previously V.P. of Engineering at Counsyl, acquired by Myriad in May 2018, which developed insurance-reimbursable genetic tests for fetal-maternal health. Scott was a graduate student in the Bader lab and developed variational optimization methods for stochastic block models of protein and genetic interactions.</p>
<p>Fun facts: Scott’s work at Counsyl is one reason why amniocentesis and chorionic villus sampling have been largely replaced by genetic analysis of circulating fetal DNA, which is easier to obtain by simple blood draws and is more reliable as well. Scott and Sarah (see below) introduced Nerf culture to the Bader lab.</p>
<h4 id="yan-qi">Yan Qi</h4>
<p><img src="/images/members/yan_qi.jpg" alt="Yan Qi"></p>
<p>Yan Qi</p>
<p>Yan Qi, Ph.D., is Senior Manager of U.S. Reporting an Analytics at Celgene, a biopharmaceutical company developing new treatments for cancer and severe disorders. Previously Yan worked in pharmaceutical consulting and at Pfizer. Yan’s research as a Ph.D. student in the Bader lab focused on genetic interactions, in which two individually neutral mutations combine to yield a strong combined effect, for example lethality. She developed new graph diffusion kernel methods, similar to Google PageRank but applicable to networks with a mix of positive (friends) and negative (enemies) interactions. While working on her Ph.D., Yan was a summer intern at IBM Research.</p>
<p>Fun facts: Yan was the first student to earn her Ph.D. in the Bader lab.</p>
<h4 id="sarah-m-richardson">Sarah M. Richardson</h4>
<p><img src="/images/members/sarah_richardson.jpg" alt="Sarah M. Richardson"></p>
<p>Sarah M. Richardson<br>
<a href="http://www.microbyre.com">http://www.microbyre.com</a></p>
<p>Sarah M. Richardson, Ph.D., is CEO of MicroByre, a synthetic biology start-up that is domesticating non-model bacterial species (basically everything other than E. coli) to create valuable genetically tractable strains. As Sarah says, we like ham and we like eggs, but this doesn’t mean that we should propose putting the egg-laying module from chickens into pigs. Unfortunately, synthetic biology has often followed this model blindly in proposing to make E. coli a general purpose chassis that will accept orthogonal modules with clean interfaces.</p>
<p>Fun facts: Sarah was Joel’s first Ph.D. student in synthetic biology. Joel’s children idolize her, for good reason.</p>
<h4 id="corban-rivera">Corban Rivera</h4>
<p><img src="/images/members/corban.jpg" alt="Corban Rivera"></p>
<p>Corban Rivera</p>
<p>Corban Rivera, Ph.D., is a Senior Data Scientist at the Applied Physics Laboratory, Johns Hopkins University. He was a postdoctoral researcher in the Bader lab working on a joint project with the Popel lab in the Department of Biomedical Engineering to identify genes involved in angiogenesis, which is important both for cardiovascular disease and for cancer.</p>
<h4 id="giovanni-stracquadanio">Giovanni Stracquadanio</h4>
<p><img src="/images/members/giovanni.jpg" alt="Giovanni Stracquadanio"></p>
<p>Giovanni Stracquadanio<br>
<a href="https://www.stracquadaniolab.org">https://www.stracquadaniolab.org</a></p>
<p>Giovanni Stracquadanio, Ph.D., is a Senior Lecturer (with tenure) in Synthetic Biology in the School of Biological Sciences at the University of Edinburgh. Giovanni develops and applies computational methods to build and analyze genomes to understand the molecular mechanisms underpinning complex phenotypes and diseases. He focuses on computer-aided engineering methods for synthetic genomics and inference methods for cancer genomics and genetics. Giovanni was a postdoctoral research in the Bader lab who worked heavily on the Saccharomyces cerevisiae (Sc2.0) project. He went into the wet-lab to learn synthetic biology hands-on and is an inventor of a new method for gene assembly.</p>
<p>Fun facts: Giovanni and his wife were students together in Sicily.</p>
<h4 id="kun-yang">Kun Yang</h4>
<p><img src="/images/members/members_kun_yang.jpg.webp" alt="Kun Yang"></p>
<p>Kun Yang (with Joel)</p>
<p>Kun Yang, Ph.D., is a Research Associate and Instructor in the Johns Hopkins Schizophrenia Center. She was a postdoctoral researcher in the Bader lab and worked extensively on the Sc2.0 project. The picture shows our relief at the low number of fitness defects in the synthetic yeast chromosomes.</p>
<p>Fun facts: Kun continues to help the Bader lab jump-start synthetic biology servers.</p>
<h4 id="ping-ye">Ping Ye</h4>
<p><img src="/images/members/ye-ping_02_web.jpg" alt="Ping Ye"></p>
<p>Ping Ye<br>
<a href="https://www.sdstate.edu/directory/ping-ye">https://www.sdstate.edu/directory/ping-ye</a></p>
<p>Ping Ye, Ph.D., is Adjunct Associate Professor at South Dakota State University and an Investigator at the Avera Research Institute. Her current work focuses on identifying the relationship between prenatal environmental exposures, epigenetic alternations, and long-term outcomes of child development and reproduction. She has very broad expertise including agent-based models, applied math, metabolic models, probabilistic graphical models, and computational genetics and genomics. In the Bader lab she worked on methods for analyzing genetic interactions leading to synthetic lethality and related phenotypes and for predicting the functional effects of protein-coding mutations. Her Ph.D. was with Prof. Denise Kirschner at the University of Michigan, where she received the MacNeal Distinguished Dissertation Award for her Ph.D. thesis.</p>
<p>Fun facts: Ping was Joel’s first lab member. With help from collaborators in the Boeke lab, she learned how to do her own experiments to introduce protein-coding mutations and assess fitness.</p>
<h4 id="jianan-zhan">Jianan Zhan</h4>
<p><img src="/images/members/jianan.jpg" alt="Jianan Zhan"></p>
<p>Jianan Zhan<br>
<a href="https://twitter.com/jiananzhan?lang=en">https://twitter.com/jiananzhan?lang=en</a></p>
<p>Jianan Zhan, Ph.D., is a scientist at 23andme, which provides genetic services directly to consumers for personalized prediction of disease risk and analysis of genetic ancestry. Jianan’s Ph.D. research in the Bader lab was to develop methods for pleiotropy, in which a single variant or gene can contribute to multiple independent phenotypes. He developed Bayesian generate models that, through permutation, yielded frequentist p-values and also suggested the genetic architecture of pleiotropic traits.</p>
<p>Fun facts: Jianan had originally intended to work in Cell and Tissue Engineering.</p>
</section>
    <section class="flex-ns flex-wrap justify-around mt5">
      
    </section></article>

    </main>
    <footer class="bg-black bottom-0 w-100 pa3" role="contentinfo">
  <div class="flex justify-between">
  <a class="f4 fw4 hover-white no-underline white-70 dn dib-ns pv2 ph3" href="https://baderzone.github.io/" >
    &copy;  Bader Lab 2022 
  </a>
    <div>
<div class="ananke-socials">
  
</div></div>
  </div>
</footer>

  </body>
</html>
